app/models/concerns/folio_json_export.rb

```diff
@@ -0,0 +1,13 @@
+module FolioJsonExport
+  def self.extended(document)
+    document.will_export_as(:folio_json, 'application/json')
+  end
+
+  def export_as_folio_json
+    {
+      "holdings_json_struct" => JSON.parse(self[:holdings_json_struct].first),
+      "items_json_struct" => JSON.parse(self[:items_json_struct].first),
```

Comment:
```suggestion
```

I've removed items_json_struct, now that we're hitting `/inventory-hierarchy/items-and-holdings` to get both the holdings and the items in one request 🤷‍♂️

Reply:
@cbeer I removed items_json_struct from this code in latest push